geti_sdk/geti.py

```diff
@@ -1220,6 +1220,7 @@ def deploy_project(
         project_name: str,
         output_folder: Optional[Union[str, os.PathLike]] = None,
         models: Optional[Sequence[BaseModel]] = None,
+        include_explainable_ai_head: bool = False,
```

Comment:
I'm not sure about the naming here, it seems a bit too long and many users may not be familiar with the `explainable ai head` concept. How about just `enable_explainable_ai` instead? 

```suggestion
        enable_explainable_ai: bool = False,
```

Reply:
Renamed!